.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,114 @@
+name: 🐛 Bug Report
+description: Submit a bug report to help us improve
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Please help us help you!
+
+        Before filing your issue, ask yourself:
+
+        - Is this clearly a Pact Python bug?
+        - Do I have basic ideas about where it goes wrong?
+        - Could it be because of my own mistakes?
+
+        **The GitHub issue tracker is not a support forum**. If you are not sure whether it could be your mistakes, ask on [Slack](https://slack.pact.io).
+
+        Make the bug obvious. Ideally, we should be able to understand it without running any code.
+
+        Bugs are fixed faster if you include:
+        - A reproduction repository to replicate the issue
+        - Pact files
+        - Logs
+
+  - type: checkboxes
+    attributes:
+      label: Have you read the Contributing Guidelines on issues?
+      options:
+        - label: I have read the [Contributing Guidelines on issues](https://github.com/pact-foundation/pact-python/main/CONTRIBUTING.md#issues).
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: Please check the following items before creating a issue. This way we know you've done these steps first.
+      options:
+        - label: I'm using the latest version of `pact-python`.
+          required: true
+        - label: I have read the console error message carefully (if applicable).
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Reproducible demo
+      description: |
+        Paste the link to an example repo if possible, and exact instructions to reproduce the issue.
+
+        > **What happens if you skip this step?** Someone will read your bug report, and maybe will be able to help you, but it's unlikely that it will get much attention from the team. Eventually, the issue will likely get closed in favor of issues that have reproducible demos.
+
+        Please remember that:
+
+        - Issues without reproducible demos have a very low priority.
+        - The person fixing the bug would have to do that anyway. Please be respectful of their time.
+        - You might figure out the issues yourself as you work on extracting it.
```

Comment:
110% agree on these, especially the 2nd! and definitely the 3rd, happened to me a fair few times. It's always nice to share back the results (which can be done via an issue) that has served as helpful do me many a time. Random comment with this is how I solved x ends up being an absolute gem that loads of people react to.

Probably indicative of that needing to be in the docs!